### PR TITLE
creating hear and headlien of portfolio website

### PR DESCRIPTION
**Pull request by ahmedadel56**
in this branch, I am trying to add a navbar and information about me

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
```markdown
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not show linter checks on PR. Please configure linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

Invalid Code Review Request **does not count** into the code reviews limit._
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio header & headline section</title>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Let us use the descriptive title for our project, for example use your full names.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  min-height: 86vh;
+  background-color: #1c1a19;
+}
+
+main .text-content {
+  margin: 30px 20px 0 30px;
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+}
```
Comment (Emmyn5600) [inline style.css]: - [ ] DRY stands for `Don't Repeat Yourself`, however you are repeating ourselves in your code, `display:flex , align-items`.

How do you keep CSS dry?

   DRY CSS comes down to 3 things:
- Group reusable css properties together.
- Name these groups logically.
- Add your selectors to the various css groups.

You can make a class and call it `.d-flex` to be called wherever you need that action

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    </header>
+    <main>
+        <section class="text-content">
+            <h1>Hey There.<br>I'm Ahmed</h1>
+            <h2 class="caption">I’am a Software Developer</h2>
+            <p>I can help you build a product , 
+                feature or website Look through some of my work
+                and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.
+            </p>
```
Comment (Emmyn5600) [inline index.html:29]: - [ ] Your code are well organized however you need to fix your headings and paragraph.
port11

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @ahmedadel56 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:

Changes required :writing_hand:  
    
    -  Kindly, use past tense for the making the descriptive PR and present tense for descriptive commits
      
   - Please check the comment under the review. 
   
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by ahmedadel56** [comment]
Please check changes and give your feedback

**Review by carlos-ssh — APPROVED** [review]
Hi @ahmedadel56 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**